CHANGELOG.md

```diff
@@ -1,3 +1,78 @@
+Version 19.3
+-------------
+19.3.1
+  - Changes:
+    - [#1497](https://github.com/huge-success/sanic/pull/1497)
+      Add support for zero-length and RFC 5987 encoded filename for
+      multipart/form-data requests.
+
+    - [#1484](https://github.com/huge-success/sanic/pull/1484)
+      The type of `expires` attribute of `sanic.cookies.Cookie` is now
+      enforced to be of type `datetime`.
+
+    - [#1482](https://github.com/huge-success/sanic/pull/1482)
+      Add support for the `stream` parameter of `sanic.Sanic.add_route()`
+      available to `sanic.Blueprint.add_route()`.
+
+    - [#1481](https://github.com/huge-success/sanic/pull/1481)
+      Accept negative values for route parameters with type `int` or `number`.
+
+    - [#1476](https://github.com/huge-success/sanic/pull/1476)
+      Deprecated the use of `sanic.request.Request.raw_args` - it has a
+      fundamental flaw in which is drops repeated query string parameters.
+      Added `sanic.request.Request.query_args` as a replacement for the
+      original use-case.
+
+    - [#1472](https://github.com/huge-success/sanic/pull/1472)
+      Remove an unwanted `None` check in Request class `repr` implementation.
+      This changes the default `repr` of a Request from `<Request>` to
+      `<Request: None />`
+
+    - [#1470](https://github.com/huge-success/sanic/pull/1470)
+      Added 2 new parameters to `sanic.app.Sanic.create_server`:
+      - `return_asyncio_server` - whether to return an asyncio.Server.
+      - `asyncio_server_kwargs` - kwargs to pass to `loop.create_server` for
```

Comment:
I'm really not sure I understand this this PR.
As far as I can tell the default is now just broken.
https://github.com/huge-success/sanic/pull/1470/files#diff-c4444a35dbd5e37ee480b0e8888e0880R1196
This changes the default from `run_async=True` to `run_async=False`
In which case... we need a known issues section?

Reply:
Take a look at the comment here: https://github.com/huge-success/sanic/pull/1470#issue-244765204

Reply:
That hasn't helped me I'm afraid. Sorry.

both of these programs run on 18.12 but error on 19.3:
```python
import asyncio, sanic

loop = asyncio.new_event_loop()
app = sanic.Sanic()
server = loop.run_until_complete(app.create_server())
```

```python
import sanic, uvloop

loop = uvloop.new_event_loop()
app = sanic.Sanic()
server = loop.run_until_complete(app.create_server())
```

I take it this is unintentional, and is a bug I should raise?